README.md

```diff
@@ -29,16 +29,23 @@ This component creates a connection to an SFTP server to read and upload files.
 The optional environment variable `MAX_FILE_SIZE` should be set in settings to provide the maximum file size that can be uploaded in **megabytes (mb)**. The default value for `MAX_FILE_SIZE` is 10MB.
 
 ## Credentials
-### User Name
-Username for SFTP server
-### Password
-Password for SFTP server
 ### Host
 Host name of SFTP server
 ### Port
 Optional, port of SFTP server. Defaults to 22 if not set.
+### User Name
+Username for SFTP server
+### Password
+Password for SFTP server.
+
+**Note**: field `Private Key` should stay empty in case you fill a password.
+### Private Key
+Your Private Key
+For accessing to secure SFTP servers configures with key-based authentication you should store your `Public key` to server (contact your server administrator) and fill this field by your `Private key`.
```

Comment:
Means?
For accessing to secure SFTP servers configure**d** with key-based authentication...